aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/PluginReflectionUtility.java

```diff
@@ -13,6 +13,26 @@
  */
 package com.exadel.aem.toolkit.core.util;
 
+import com.exadel.aem.toolkit.api.annotations.main.ClassField;
```

Comment:
Just for the case, please observe formatting rules existing for the project. I know it's the IDE's autoformat reshuffling imports for you, But that only means, when anoter team member corrects a sngle character in the class, the whole lot of imports gets re-shuffled back. That produces a lot of meaningles changes in the commits. So please be sure to exclude from your commits <changes that don't actually change anything>
This is before we introduce a tool for distributing formats across the team. I know we must do it)